WordPress/WordPressTest/ReaderPostCellActionsTests.swift

```diff
@@ -0,0 +1,106 @@
+import XCTest
+
+@testable import WordPress
+
+final class ReaderPostCellActionsTests: CoreDataTestCase {
```

Comment:
I added 2 unit tests to verify the presence of the Report / Block actions when the actions sheet is presented.